### PR TITLE
CommonJS-only setup

### PR DESCRIPTION
**Pull request by PaulLeCam**
## Description

I think the issue with CommonJS imports in https://github.com/ceramicstudio/self.id/issues/53 is due to having dual CJS and ESM exports. As 3ID Connect only exports CJS, imports from ESM versions of the consuming packages fail.

The migration to ESM across the stack is ongoing but not ready yet, so in the meantime the alternative would be to go CJS-only to avoid this situation.

This mostly affects the image imports in multiauth, that require extra configuration now when using Next.js.

I also moved the `vite-basic` template and removed it from the list in the README, as it only supports ESM.

## How Has This Been Tested?

- Updated unit tests
- Run self.id app
- Run templates

### CODE REVIEW

File: apps/developers/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
-    "build": "docusaurus build",
+    "--build": "docusaurus build",
```
Comment (oed) [inline apps/developers/package.json:8]: Why this change?

File: templates/webpack-basic-typescript/package.json [hunk 2]
```diff
@@ -1,31 +1,33 @@
 {
   "name": "self.id-template-webpack-basic-typescript",
-  "version": "0.1.0",
+  "version": "0.0.0",
```
Comment (oed) [inline templates/webpack-basic-typescript/package.json:3]: Why does this change from `0.1.0`to `0.0.0`?

File: templates/webpack-basic/package.json [hunk 3]
```diff
@@ -1,27 +1,29 @@
 {
   "name": "self.id-template-webpack-basic",
-  "version": "0.1.0",
+  "version": "0.0.0",
```
Comment (oed) [inline templates/webpack-basic/package.json:3]: Same thing here?

**Review by oed — COMMENTED** [review]
looks mostly good. Just a question around package versions.

File: apps/developers/package.json (hunk 1 above)
Comment (PaulLeCam) [reply apps/developers/package.json:8]: To disable it, there seems to be some issues with the Docusaurus build but as we're not currently using this, I don't think it's worth spending time fixing.

File: templates/webpack-basic-typescript/package.json (hunk 2 above)
Comment (PaulLeCam) [reply templates/webpack-basic-typescript/package.json:3]: Just for consistency with the other templates, the version doesn't matter here, we just need to set one to avoid having npm warn about it

**Review by oed — APPROVED** [review]
